db/seeds.rb

```diff
@@ -8,8 +8,12 @@
 require "open-uri"
 
 puts 'Cleaning the database...'
+Group.destroy_all
```

Comment:
I added dependent: :destroy to user.rb about teachers and students, so we don't need to destroy the teachers and students here anymore, it will be automatically. I'd put dependent: :destroy in teacher.rb for the groups as well and remove the Group.destroy_all

Reply:
ok cool so you will resolve the conflict so your code stays in the seed, right? or do I need to change something?

Reply:
ya good call, we can cahnge it while resolving